lectures/linear_equations.md

```diff
@@ -141,10 +141,12 @@ column vectors.
 
 The set of all $n$-vectors is denoted by $\mathbb R^n$.
 
-For example, 
+```{prf:example}
+:label: ex_dim
```

Comment:
```suggestion
:label: le_ex_dim
```

Reply:
@longye-tian labels in jupyter-book are global so it can be helpful to prepend a lecture identifier such as `le`

Reply:
Sounds great! I will revise the labels.